lib/assets/stylesheets/quill/quill.snow.css

```diff
@@ -850,6 +850,7 @@
   box-sizing: border-box;
   font-family: 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
   padding: 8px;
+  background: white;
```

Comment:
Please never edit the original code in the lib folder unless there is no other choice, if you want to add extra feature like this, can just add it to CSS file in the app/assets